README.md

```diff
@@ -79,8 +85,13 @@ users = [
 first_order_for_each_user = []
 
 ```
+<!-- not sure what  is the problem it gives me nothing-->
 
-### Answer
+first_order_for_each_user = []
+users.map do |order|
+  first_order_for_each_user.push("#{order[:'a bike'] } #{order[:'bees'] } " )
+end
+puts first_order_for_each_user
```

Comment:
when you use `users.map` it loops through the users so it accesses the entire user hash not just the orders
this is  an example of what you might get for each iteration 
```ruby
{
      name: 'Turki',
      orders: [
          {
              description: 'a MacBook'
          },
          {
              description: 'The West Wing DVDs'
          },
          {
              description: 'headphones'
          },
          {
              description: 'a kitten'
          }
      ]
  }
```

You can access your orders array using the key [:orders], `.first` is a good way to get the first element of the array

```ruby
first_order_for_each_user = users.map do |user|
user[:orders].first
end
p first_order_for_each_user
```